editor/src/clj/internal/node.clj

```diff
@@ -1017,7 +1017,7 @@
 
 (defn- update-fn-maps [tree f]
   (-> tree
-      (update :output (partial util/map-vals f))
+      (update :output (partial coll/map-vals f))
```

Comment:
If `coll` was a first argument to `map-vals` (like in `clojure.core/update-vals`), we wouldn't need to create a throw-away partial function; instead, we could use `(update :output coll/map-vals f)`